### ISSUE TITLE
Gradient_norm becomes NaN during 7B model training

### ISSUE THREAD

**Issue by QihangCao** [LaVi-Lab/VG-LLM#45]
{'loss': 14.6804, 'grad_norm': 86.92993927001953, 'learning_rate': 3.3333333333333333e-06, 'epoch': 0.01}
{'loss': 14.3597, 'grad_norm': 90.59915161132812, 'learning_rate': 4.000000000000001e-06, 'epoch': 0.01}
  1%|          | 60/4967 [11:52<14:16:32, 10.47s/it]
  1%|          | 61/4967 [12:03<14:33:15, 10.68s/it]
  1%|          | 62/4967 [12:16<15:34:33, 11.43s/it]
  1%|▏         | 63/4967 [12:27<15:27:43, 11.35s/it]
  1%|▏         | 64/4967 [12:42<16:45:52, 12.31s/it]
  1%|▏         | 65/4967 [12:53<16:14:37, 11.93s/it]
  1%|▏         | 66/4967 [13:02<15:09:40, 11.14s/it]
  1%|▏         | 67/4967 [13:15<15:50:58, 11.64s/it]
  1%|▏         | 68/4967 [13:28<16:18:23, 11.98s/it]
  1%|▏         | 69/4967 [13:39<15:58:00, 11.74s/it]
  1%|▏         | 70/4967 [13:53<17:02:36, 12.53s/it]                                                 
{'loss': 8195.7664, 'grad_norm': nan, 'learning_rate': 4.666666666666667e-06, 'epoch': 0.01}

When training vgllm with Qwen2.5VL-7B, the training process becomes unstable after a few iterations. Specifically, the loss value suddenly spikes to an extremely large number, and the gradient_norm turns into NaN. Has anyone else run into this problem? Any ideas on how to fix it?
